frontend/dockerfile/builder/build.go

```diff
@@ -59,6 +63,10 @@ func Build(ctx context.Context, c client.Client) (_ *client.Result, err error) {
 			}
 			return res, err
 		} else if ref, cmdline, loc, ok := parser.DetectSyntax(src.Data); ok {
+			if runtime.GOOS == "windows" {
+				_ = os.Setenv("FrontendGRPCPipe", `\\.\pipe\buildkit-frontend-bridge`+uuid.NewString())
```

Comment:
creating a new uuid here may not be necessary, there should be a session Id that is unique to each build that we can leverage for this same purpose and would navigate the risk of the `FrontendGRPCPipe` env being overwritten